pkg/controller/directvolumemigration/descriptions.go

```diff
@@ -1,24 +1,28 @@
-	DestinationNamespacesCreated:    "Checking if the destination namespace is created, currently it is a no-op",
-	CreateDestinationPVCs:           "Creating PVCs to be migrated on the destination namespace, no-op if they exist",
-	DestinationPVCsCreated:          "Check if all the PVCs that were created are Bound, currently a no-op",
-	CreateRsyncRoute:                "Creating one route per namespace for rsync on the destination cluster",
-	CreateRsyncConfig:               "Creating configmap and secrets on both source and destination clusters for rsync configuration",
-	CreateStunnelConfig:             "Creating configmap and secrets for stunnel certs used to connect to rsync on both source and destionation cluster",
-	CreatePVProgressCRs:             "Creating direct volume migration progress CR to get progress percent and transfer rate",
-	CreateRsyncTransferPods:         "Creating rsync daemon pods on the destination cluster",
-	WaitForRsyncTransferPodsRunning: "Wait for the rsync daemon pod to be running",
-	CreateStunnelClientPods:         "Create stunnel client pods on the source cluster",
-	WaitForStunnelClientPodsRunning: "Waiting for the stunnel client pods to be running",
-	CreateRsyncClientPods:           "Create Rsync client pods",
-	WaitForRsyncClientPodsCompleted: "Wait for the rsync client pods to be completed",
-	DeleteRsyncResources:            "Delete all the resources created while running this migration",
-	MigrationFailed:                 "The migration attempt failed, please see errors for more details",
-	Completed:                       "Complete",
+	Created:                              "DVM CR has been created",
+	Started:                              "DVM Controller is configuring DVM CR",
+	Prepare:                              "DVM Controller is preparing the environment for volume migration. Currently it is non-operational",
+	CleanStaleRsyncResources:             "Cleaning up stale resources from previous migrations",
+	WaitForStaleRsyncResourcesTerminated: "Waiting for stale resources to terminate",
+	CreateDestinationNamespaces:          "Creating target namespaces",
+	DestinationNamespacesCreated:         "Checking if the target namespaces have been created, currently it is non-operational",
+	CreateDestinationPVCs:                "Creating PVCs in the target namespaces",
+	DestinationPVCsCreated:               "Checking whether the created PVCs are bound",
+	CreateRsyncRoute:                     "Creating one route for each namespace for Rsync on the target cluster",
+	CreateRsyncConfig:                    "Creating a config map and secrets on both the source and target clusters for Rsync configuration",
+	CreateStunnelConfig:                  "Creating a config map and secrets for Stunnel to connect to Rsync on the source and target clusters",
+	CreatePVProgressCRs:                  "Creating a Direct Volume Migration Progress CR to get progress percent and transfer rate",
```

Comment:
```suggestion
	CreatePVProgressCRs:                  "Creating a Direct Volume Migration Progress CR to get progress percentage and transfer rate",
``

Reply:
nit: should above be percentage instead? I remember Dylan had a similar comment on CRD field

Reply:
Ah makes sense, I'll update